File: Api.Repository.Models/Person.cs
using Api.Common.Enums;
using Api.Repository.Models.Base;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Api.Repository.Models
{
    public class Person : BaseModel, IEntity
    {
        public string Name { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public bool Active { get; set; }
        public PersonTypeEnum PersonType { get; set; }
        public User User { get; set; }
    }
}

File: Web/src/app/models/person.ts
import { PersonTypeEnum } from '../enums/person-type.enum';

export interface PersonModel {
    id: number;
    garageId: number;
    name: string;
    lastName: string;
    phoneNumber: string;
    email: string;
    active: boolean;
    personType: PersonTypeEnum;
}
